ontolearner/learner/llm.py

```diff
@@ -1,12 +1,29 @@
-from ..base import AutoLearnerLLM
-from typing import Any, List
+from ..base import AutoLLM
+from typing import List
+from transformers import AutoModelForCausalLM, AutoTokenizer
+import torch
 
-class AutoLLM(AutoLearnerLLM):
+
+class AutoLearnerLLM(AutoLLM):
     def __init__(self):
         super().__init__()
 
-    def load(self, model_id:str):
-        pass
+    def load(self, model_id: str = "mistralai/Mistral-7B-Instruct-v0.1"):
```

Comment:
What about the token? If I use the llama model that asks me a Huggingface token, what will happen here? so I would recommend considering a token as a kind of variable that is set to `token:str=' '"` in the init function of this model. And just pass it to the tokenizer and automodelforcausallm they will operate if token is just a empty string!